gradle/mockito-core/javadoc.gradle

```diff
@@ -38,8 +38,8 @@ javadoc {
     options.noIndex = false
     options.noNavBar = false
     options.noTree = false
-    options.links = ['https://docs.oracle.com/javase/6/docs/api/',
-                     'https://junit.org/junit4/javadoc/4.12/']
+    options.links('https://junit.org/junit4/javadoc/4.12/')
+    options.linksOffline('https://docs.oracle.com/javase/6/docs/api/', "$buildDir/javase-6-docs/")
```

Comment:
Nit: while we are at it, let's update our Javadoc version to 8 as well, since that's the version we support